api/tests/certification/complementary-certification/unit/domain/usecases/attach-badges_test.js

```diff
@@ -145,20 +155,18 @@ describe('Unit | UseCase | attach-badges', function () {
               { badgeId: 1, level: 1 },
               { badgeId: 2, level: 2 },
             ],
-            complementaryCertificationRepository,
             badgeRepository,
           });
 
           // then
-          expect(complementaryCertificationRepository.getById).to.have.been.calledOnce;
           expect(error).to.be.instanceOf(NotFoundError);
           expect(error.message).to.equal("One or several badges don't exist.");
         });
       });
     });
   });
 
-  context('when complementary certification does exist and levels checks are ok', function () {
+  context('when levels checks are ok', function () {
```

Comment:
Est-ce que si on a ce context on ne devrait pas avoir par ailleurs 'when levels checks are not ok'